relay-server/src/actors/processor.rs

```diff
@@ -1639,6 +1639,20 @@ impl EnvelopeProcessorService {
                     sdk = envelope.meta().client_name().unwrap_or("proprietary"),
                     platform = event.platform.as_str().unwrap_or("other"),
                 );
+
+                let otel_context = event
+                    .contexts
+                    .value()
+                    .and_then(|contexts| contexts.get("otel"))
+                    .and_then(Annotated::value);
+
+                if otel_context.is_some() {
+                    metric!(
```

Comment:
How many OTel events (and metrics) are we expecting, and how big is the cardinality of these counters (are most events coming from the same SDK) are in the statsd client in relay? If the number is not large it's ok. My concern is that we've been in the maximum threshold before, and the additional metrics are dropped.

Reply:
We should be consuming at the same volume as transaction source information, which is set above in `RelayCounters::EventTransactionSource` with the same cardinality, and we've been able to manage that fine.

Most events should come from 4 SDKs atm (Node, Java, Python, Ruby), so it's even less than transaction source at the moment.